### ISSUE TITLE
Linked 2D orthogonal views of 3D data

### ISSUE THREAD

**Issue by raacampbell** [napari/napari#662]
## 🚀 Feature
It would be great if napari had the ability to show data as linked orthogonal views. e.g. Clicking or dragging across the current coronal plane causes the image in the sagittal and horizontal planes to update. 

## Motivation
Fiji and Icy both have an orthogonal views feature. At the SWC we wrote an orthogonal views Python-based viewer, [Lasagna](https://github.com/SainsburyWellcomeCentre/lasagna), for visualising registration to the Allen Atlas. It's a very convenient way to navigate the data. It overlays traced neurons Allen brain areas, etc, but becomes slow for datasets over about 5 GB. We would likely move to napari if it supported linked orthogonal views.

**Comment by jni** [comment]
Thanks @raacampbell! This is definitely on our to-do list, but it appears we didn't have an issue for it!

**Comment by sofroniewn** [comment]
Yes, we first need to add support multiple canvases each with their own cameras, and then need to set up various types of linking. Orthogonal views is one you mention (possibly with the full 3D view in the remaining corner). Another is linked 2D views in a grid mode for zooming into the same spot. This was requested in #565, where we quickly implemented a grid view with just one camera, but where we still need to implement the linked camera version

**Comment by sofroniewn** [comment]
Also @raacampbell if you're interesting in contributing around this feature, or any of the others on our [`0.2`](https://github.com/napari/napari/issues/301) and [`0.3`](https://github.com/napari/napari/issues/420) milestones (or features that should be added to those milestones) let us know. We'd love to work with you. Lasagna is very impressive, and I'm sure you learnt a lot during that process that's very relevant here.

We're also planning to build a plugin ecosystem around napari where more domain specific analysis tools can live and be used from within the viewer - see #263 for some really early ideas around that - and would love your input there too

**Comment by raacampbell** [comment]
I would love to but right now I'm overwhelmed with coding and construction on some hardware projects. I'm hoping to slowly transition back to analysis, though, so contributing to napari is something I would like to do. I don't see the sense in developing multiple very similar solutions to the same problem and you have something much more mature and interesting than I do.

**Comment by psobolewskiPhD** [comment]
Bumping this as it came up in the ome-zarr community call

Similar/same plugin issue:
https://github.com/napari/napari/issues/1478

Current best ortho-viewer plugin
https://github.com/AnniekStok/napari-orthogonal-views
